src/routes/study-space/+page.svelte

```
@@ -0,0 +1,765 @@
+								{#if message.role === 'assistant'}
+									<div class="w-8 h-8 rounded-xl bg-gradient-to-br from-primary/20 to-accent/20 flex items-center justify-center shrink-0 mt-1">
+										<Sparkles size={14} class="text-primary" />
+									</div>
+								{/if}
+
+								<div class="flex-1 max-w-[85%] {message.role === 'user' ? 'ml-auto' : ''}">
+									<div class="rounded-2xl px-4 py-3 {message.role === 'user'
+										? 'bg-primary text-primary-content rounded-br-md'
+										: 'bg-base-100 border border-base-300/60 shadow-xs rounded-bl-md'}">
+										<div class="text-sm leading-relaxed whitespace-pre-line {message.role === 'assistant' ? 'chat-content' : ''}">
+											{@html message.content.replace(/\*\*(.*?)\*\*/g, '<strong>$1</strong>').replace(/\n/g, '<br />')}
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)